src/components/CompareResults/RevisionRow.tsx

```diff
@@ -102,6 +104,28 @@ const stylesLight = {
       '.expand-button': {
         backgroundColor: Colors.Background300,
       },
+      '.status-hint': {
+        display: 'inline-flex',
+        gap: '6px',
+        borderRadius: '4px',
+        padding: '4px 10px',
+      },
+      '.status-hint.regression': {
+        backgroundColor: '#FFE8E8',
+      },
+      '.status-hint.improvement': {
+        backgroundColor: '#D8EEDC',
+      },
+      '.status-hint .MuiSvgIcon-root': {
+        height: '16px',
+      },
+      '.status-hint.improvement .MuiSvgIcon-root': {
+        color: '#017A40',
+      },
+      '.status-hint.regression .MuiSvgIcon-root': {
+        marginTop: '2px',
```

Comment:
can you please add a comment for this marginTop? Eg `// we need to move the icon a bit lower so that it _looks_ center`
(same below)